tests/repository_simulator.py

```diff
@@ -136,6 +139,7 @@ def targets(self) -> Targets:
         return self.md_targets.signed
 
     def all_targets(self) -> Iterator[Tuple[str, Targets]]:
+        """Yield role name and signed portion of targets one by one"""
```

Comment:
I am not sure if relevant comment.  
To be consistent with another docstring you had added, finish with a period.
If it makes sense, it would be interesting to apply the other changes that are part of this PR.

Reply:
I will add `.` at the end of the docstrings.
Do you have a better suggestion for this docstring? 
The function is simple and I am not sure what else can I write here.

Reply:
I don't have a better suggestion for this docstring. LGTM

I think what we are doing in the ``ngclient`` for docstring is the best option. In a sense, it can become good documentation in the end (as part of readthedocs/sphinx).
I understand that this is a test part, but it can help a lot of new contributors. Any thoughts @jku ?